.github/workflows/build-iso.yml

```
@@ -49,11 +43,17 @@
         image_tag: ${{ env.IMAGE_TAG }}
         version: ${{ env.VERSION }}
         variant: ${{ env.VARIANT }}
-        iso_name: ${{ env.IMAGE_NAME }}-v${{ steps.iso_version.outputs.version }}.iso
+        iso_name: ${{ env.IMAGE_NAME }}-${{ env.VERSION }}.iso
         flatpak_remote_name: flathub
         flatpak_remote_url: https://flathub.org/repo/flathub.flatpakrepo
         flatpak_remote_refs_dir: /github/workspace/flatpak_refs
 
+    - name: Get ISO version
+      id: iso_version
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:32: Double quote to prevent globbing and word splitting

(shellcheck)